src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -203,11 +203,15 @@ func (s *cloudService) CreateVM(ctx context.Context, req *pb.CreateVMRequest) (r
 	// Get Pod VM cpu and memory from annotations
 	vcpus, memory := util.GetCPUAndMemoryFromAnnotation(req.Annotations)
 
+	// Get Pod VM image from annotations
+	image := util.GetImageFromAnnotation(req.Annotations)
```

Comment:
Can the generic changes be in a separate commit to distinguish from provider specific implementation ?